.github/workflows/auto_request_review.yml

```diff
@@ -0,0 +1,16 @@
+name: Auto Request Review
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, reopened]
+
+jobs:
+  auto-request-review:
+    name: Auto Request Review
+    permissions: write-all
```

Comment:
Currently trying to figure out if I can narrow the permissions to least privilege and ensure that the job passes: https://github.com/cncf/tag-security/runs/7736402648?check_suite_focus=true